### ISSUE TITLE
Redirection Error

### ISSUE THREAD

**Issue by bperniciaro** [johngodley/redirection#694]
```
Plugin: 3.1
WordPress: 4.9.2 (single)
PHP: 5.6.27
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36
REST API: https://www.cheatsheetwarroom.com/blog2/wp-json/

Error: Failed to fetch

Error: Failed to fetch
```

**Comment by johngodley** [comment]
What is the URL for your site? When I try the one above it returns an error that the connection was reset

**Comment by bperniciaro** [comment]
Sorry.  You can see my default page here: http://www.cheatsheetwarroom.com/blog2/

I added the Pluin, added a few redirects, and never had a problem.  I opened it today and saw the error:

Plugin: 3.1
WordPress: 4.9.2 (single)
PHP: 5.6.27
Browser: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36
REST API: https://www.cheatsheetwarroom.com/blog2/wp-json/

Error: Failed to fetch

**Comment by johngodley** [comment]
This should be fixed in version 3.1.1
